drivers/console/multi_console.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <assert.h>
+#include <drivers/console.h>
+
+console_t *console_list;
+uint8_t console_state = CONSOLE_FLAG_BOOT;
```

Comment:
``console_list`` and ``console_state`` can be static.

Reply:
No, they need to be exported to be accessible by crash_console_helpers.S.